packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectIModel.tsx

```diff
@@ -29,41 +30,52 @@ interface SelectIModelProps extends IModelGridProps {
   iTwinName?: string;
 }
 
+// Check if the IModel is downloaded by the user recently and remove from recent if the file doesn't exist anymore
```

Comment:
idk if this makes sense anymore.
you cant get to this pg without going through the home pg, which will remove missing files

Reply:
Hmmmm the homepage doesn't remove missing files though. For the Recents component inside Home page, it only removes when you click on the missing file. I'm not sure if making the Recents component to check all files in recents settings and remove them? Since then the missing files will never show up in the Recents component.

Reply:
maybe it makes more sense to filter out and remove all misisng files on the backend before you send it to the frontend?

Reply:
whats better ux? 
seeing a missing file, trying to click on it and then removing it?
or not showing it at all?
or showing it as disabled so you cant click on it?